suse2013/common/trim-verbatim.xsl

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Purpose:
+     Trim start/end of verbatim areas (screen, programlisting, synopsis)
+
+
+    Author:     Stefan Knorr <sknorr@suse.de>
+    Copyright:  2015
+
+-->
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:template match="programlisting/text()|screen/text()|synopsis/text()">
```

Comment:
Maybe it's worth to split the template into `programlisting/text()[1]` and `programlisting/text()[last()]`?

Not sure if this helps. Could probably problematic when the first node is a element node.